decidim-core/app/packs/src/decidim/editor/extensions/decidim_kit/index.js

```diff
@@ -3,12 +3,11 @@ import { Extension } from "@tiptap/core";
 import StarterKit from "@tiptap/starter-kit";
 import CodeBlock from "@tiptap/extension-code-block";
 import Underline from "@tiptap/extension-underline";
+import Heading from "@tiptap/extension-heading";
```

Comment:
I think we need to add Mention as well 

```suggestion
import Heading from "@tiptap/extension-heading";
import Mention from "@tiptap/extension-mention";
```